### ISSUE TITLE
Fix CI

### ISSUE THREAD

**Issue by mrakitin** [NSLS-II/sirepo-bluesky#17]
The following temporary command, compatible with the one from `README.md` (but slightly updated for testing purposes) can be used for now in CI to unblock it:
```bash
$ docker run -d -t --rm -e SIREPO_AUTH_METHODS=bluesky:guest -e SIREPO_AUTH_BLUESKY_SECRET=bluesky -e SIREPO_SRDB_ROOT=/sirepo -e SIREPO_COOKIE_IS_SECURE=false -p 8000:8000 -v $PWD/sirepo_bluesky/tests/SIREPO_SRDB_ROOT:/SIREPO_SRDB_ROOT:ro,z radiasoft/sirepo:20200220.135917 bash -l -c "mkdir -v -p /sirepo/user/ && cp -Rv /SIREPO_SRDB_ROOT/* /sirepo/user/ && sirepo service http"
```

**Comment by mrakitin** [comment]
This command worked great for me based on the recommendation from @e-carlin in https://github.com/radiasoft/sirepo/issues/3486#issuecomment-815330855:

```bash
$ docker run -d -t --init --rm -u vagrant --name sirepo -e SIREPO_AUTH_METHODS=bluesky:guest -e SIREPO_AUTH_BLUESKY_SECRET=bluesky -e SIREPO_SRDB_ROOT=/sirepo -e SIREPO_COOKIE_IS_SECURE=false -p 8000:8000 -v $PWD/sirepo_bluesky/tests/SIREPO_SRDB_ROOT:/SIREPO_SRDB_ROOT:ro,z radiasoft/sirepo:beta bash -l -c "mkdir -v -p /sirepo/user/ && cp -Rv /SIREPO_SRDB_ROOT/* /sirepo/user/ && sirepo service http"
```

**Comment by mrakitin** [comment]
@jklynch, do you mind trying the command above which uses `radiasoft/sirepo:beta` tag?

**Comment by mrakitin** [comment]
Resolved via #15.
